backend/app/controllers/internal/companies/dividend_computations_controller.rb

```
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+class Internal::Companies::DividendComputationsController < Internal::Companies::BaseController
+  before_action :set_dividend_computation, only: [:show, :confirm]
+
+  def index
+    authorize DividendComputation
+    computations = Current.company.dividend_computations.includes(:dividend_computation_outputs)
+    render json: { 
```

RuboCop (1.75.5)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)